yk_utils/apis/face_authentication.py

```diff
@@ -4,21 +4,35 @@
 from bs4 import BeautifulSoup
 
 
+class FaceAuthenticationResult:
```

Comment:
Perhaps we can leave this name more generic, since it doesn't necessarily has a face authentication dependency. Like "ApiResult" or "ApiResponse", hope you have a better name idea if you find this point of view true :)

Reply:
Will change it